src/main/java/io/codelex/exceptions/practice/Exercise2.java

```diff
@@ -0,0 +1,31 @@
+package io.codelex.exceptions.practice;
+
+public class Exercise2 {
```

Comment:
`After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller.`